sw/device/silicon_creator/lib/drivers/pinmux.c

```diff
@@ -47,32 +30,60 @@ typedef struct pinmux_output {
 } pinmux_output_t;
 
 /**
- * MIO pad to peripheral output mappings.
+ * UART RX pin.
  */
-static const pinmux_output_t kPinmuxOutputs[] = {
-    /**
-     * UART
-     */
-    {.pad = kTopEarlgreyPinmuxMioOutIoc4,
-     .periph = kTopEarlgreyPinmuxOutselUart0Tx},
+static const pinmux_input_t kInputUart = {
+    .periph = kTopEarlgreyPinmuxPeripheralInUart0Rx,
+    .pad = kTopEarlgreyPinmuxInselIoc3,
 };
 
+/**
+ * Bootstrap pin.
+ */
+static const pinmux_input_t kInputBootstrap = {
+    .periph = kTopEarlgreyPinmuxPeripheralInGpioGpio17,
+    .pad = kTopEarlgreyPinmuxInselIob8,
+};
```

Comment:
TODO: the real chip will have 3 strapping pins encoding 64-possible values by way of weak and strong pull up/down resistors.  Configure the 3 pins as inputs.

Reply:
Created #11934